autoload/ale/fixers/gomod.vim

```diff
@@ -0,0 +1,9 @@
+function! ale#fixers#gomod#Fix(buffer) abort
+    return {
+    \   'command':
+    \       . ' go mod edit -fmt'
+    \       . ' %t',
+    \   'read_temporary_file': 1,
+    \}
+endfunction
+
```

Comment:
Add a test for this function and remove the extra blank line at the end of the file.